app/api/events.py

```diff
@@ -154,19 +154,26 @@ def before_post(self, args, kwargs, data=None):
 
     def after_create_object(self, event, data, view_kwargs):
         """
-        after create method to save roles for users and add the user as an accepted role(organizer)
+        after create method to save roles for users and add the user as an accepted role(owner and organizer)
         :param event:
         :param data:
         :param view_kwargs:
         :return:
         """
-        role = Role.query.filter_by(name=ORGANIZER).first()
         user = User.query.filter_by(id=view_kwargs['user_id']).first()
+        role = Role.query.filter_by(name=OWNER).first()
```

Comment:
@shreyanshdwivedi after event creation, shouldn't there be a role for `organiser` too for the owner?

Reply:
@uds5501 when event is created the user creating the event will be the owner. 
He can then send role-invites to user to become organizer.

Reply:
Will the owner have same/more permissions than an organizer? If not, fix it

Reply:
For now owner will have same permissions as that of an organizer. Later we can give more permissions to owner